base_optional_quick_create/model.py

```diff
@@ -50,14 +48,18 @@ def _register_hook(self, cr, ids=None):
                     model_obj.check_quick_create = True
         return True
 
+    def _register_hook(self, cr):
```

Comment:
Do you have a suggestion about how to do it in this case? (Where ```_register_hook``` has 2 parameters and ```_patch_quick_create``` 3